# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../netkit-telnet/line-name.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./telnetd/utility.c~	Sun Dec 12 15:59:45 1999
+++ ./telnetd/utility.c	Sat Apr  8 11:04:09 2000
@@ -467,11 +467,10 @@
 		switch (*++cp) {
 
 		case 't':
-			slash = strrchr(line, '/');
-			if (slash == NULL)
+			if ( strncmp(line,"/dev/",5) )
 				putstr(line);
 			else
-				putstr(slash+1);
+				putstr(line+5);
 			break;
 
 		case 'h':
